src/DssSpell.t.base.sol

```diff
@@ -1905,6 +1974,140 @@ contract DssSpellTestBase is Config, DssTest {
         assertEq(litePsm.tin(),  type(uint256).max, _concat("checkLitePsmIlkIntegration/mom-halt-invalid-tin-",  p.ilk));
         assertEq(litePsm.tout(), type(uint256).max, _concat("checkLitePsmIlkIntegration/mom-halt-invalid-tout-", p.ilk));
     }
+    struct AllocatorIntegrationParams {
+        bytes32 ilk;
+        address pip;
+        address registry;
+        address roles;
+        address buffer;
+        address vault;
+        address allocatorProxy;
+    }
+
+    function _checkAllocatorIntegration(AllocatorIntegrationParams memory p) internal {
+        (, uint256 rate, uint256 spot,,) = vat.ilks(p.ilk);
+        assertEq(rate, RAY);
+        assertEq(spot, 10**18 * RAY * 10**9 / spotter.par());
+
+        (address pip,) = spotter.ilks(p.ilk);
+        assertEq(pip, p.pip);
+
+        assertEq(vat.gem(p.ilk, p.vault), 0);
+        (uint256 ink, uint256 art) = vat.urns(p.ilk, p.vault);
+        assertEq(ink, 1_000_000_000_000 * WAD);
+        assertEq(art, 0);
+
+        assertEq(AllocatorRegistryLike(p.registry).buffers(p.ilk), p.buffer);
+        assertEq(address(AllocatorVaultLike(p.vault).jug()), address(jug));
+
+        assertEq(usds.allowance(p.buffer, p.vault), type(uint256).max);
+
+        assertEq(AllocatorRolesLike(p.roles).ilkAdmins(p.ilk), p.allocatorProxy);
+
+        assertEq(AllocatorVaultLike(p.vault).wards(pauseProxy),  0);
+        assertEq(AllocatorVaultLike(p.vault).wards(p.allocatorProxy), 1);
+
+        assertEq(WardsAbstract(p.buffer).wards(pauseProxy),  0);
+        assertEq(WardsAbstract(p.buffer).wards(p.allocatorProxy), 1);
+
+        assertEq(reg.join(p.ilk),   address(0));
+        assertEq(reg.gem(p.ilk),    address(0));
+        assertEq(reg.dec(p.ilk),    0);
+        assertEq(reg.class(p.ilk),  5);
+        assertEq(reg.pip(p.ilk),    p.pip);
+        assertEq(reg.xlip(p.ilk),   address(0));
+        assertEq(reg.name(p.ilk),   _bytes32ToString(p.ilk));
+        assertEq(reg.symbol(p.ilk), _bytes32ToString(p.ilk));
+
+        // Draw & Wipe from Vault
+        vm.prank(address(p.allocatorProxy));
+        AllocatorVaultLike(p.vault).draw(1_000 * WAD);
+        assertEq(usds.balanceOf(p.buffer), 1_000 * WAD);
+
+        vm.warp(block.timestamp + 1);
+        jug.drip(p.ilk);
+
+        vm.prank(address(p.allocatorProxy));
+        AllocatorVaultLike(p.vault).wipe(1_000 * WAD);
+        assertEq(usds.balanceOf(p.buffer), 0);
+    }
+
+        struct BaseTokenBridgeParams {
+        address l2bridge;
+        address l1bridge;
+        address l1escrow;
+        address[] tokens;
+        address[] l2tokens;
```

Comment:
```suggestion
        address l2Bridge;
        address l1Bridge;
        address l1Escrow;
        address[] tokens;
        address[] l2Tokens;
```
I think this can help improving the readability